scripts/opam-release.sh

```diff
@@ -18,6 +21,13 @@ if ! echo "$GIT_VERSION" | egrep -q '^v[0-9]+.[0-9]+.[0-9]+$'; then
     DEV=~dev
 fi
 
+WERROR=$(git grep -c ' -Werror' Makefile.common | cut -d ':' -f 2)
+if [ "$WERROR" -gt 0 ]; then
+    echo "WARNING: There are occurences of '-Werror' in 'Makefile.common'."
+    echo "WARNING: This is almost certainly not what you want."
+    exit 1
```

Comment:
LGTM. To be nitpicky, as the script exits with an error code, maybe echo _ERROR:_ instead of _WARNING:_?

Reply:
thanks, addressed in [f861e0a](https://github.com/Solo5/solo5/pull/565/commits/f861e0a5b2e037adbc8d6ee2ffc06fc1a38101e8)